components/ConversionForm/ConversionForm.js

```diff
@@ -3,11 +3,12 @@ import { OverlayTrigger, Tooltip } from 'react-bootstrap'
 import styled from 'styled-components'
```

Comment:
Nitpick: wondering if we should rename this as `ConvertForm` (or `ConvertStepper` as `ConversionStepper`) to be consistent in tense.